src/app.js

```diff
@@ -0,0 +1,53 @@
+let list = []
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
let list = [];
```